src/frontend/apps/drive/package.json

```diff
@@ -10,7 +10,7 @@
     "build-theme": "cunningham -g css,scss -o src/styles && mv src/styles/cunningham-tokens.scss src/styles/cunningham-tokens-sass.scss"
   },
   "dependencies": {
-    "@gouvfr-lasuite/ui-kit": "0.4.0",
+    "@gouvfr-lasuite/ui-kit": "0.4.1",
```

Comment:
We get the workspace menu when clicking on "..." of a folder

<img width="616" alt="Capture d’écran 2025-04-25 à 14 38 32" src="https://github.com/user-attachments/assets/fc8c3d14-c167-4f76-8506-40659cdcb0ca" />